apps/entity-service/src/entities/processors/entity-processor.ts

```diff
@@ -87,12 +86,31 @@ export abstract class EntityProcessor<
 
   async getLightDtos(models: ModelType[]): Promise<DtoResult<LightDto>[]> {
     const results: DtoResult<LightDto>[] = [];
+
+    let associateData: Record<number, LightDto> = {};
+
+    const allowedTypes = [Project];
+    if (allowedTypes.some(type => models[0] instanceof type)) {
+      associateData = (await this.loadAssociationData(models.map(m => String(m.id)))) as Record<number, LightDto>;
+    }
```

Comment:
Instead of this `allowedTypes` structure, let's have the default implementation just return an empty map instead of throwing an error. The base entity processor should not be responsible for knowing which subclasses have a real implementation.